pom.xml

```diff
@@ -54,7 +54,7 @@
     <msgpack.version>0.6.7</msgpack.version>
     <super-csv.version>2.1.0</super-csv.version>
     <td-client.version>0.5.5</td-client.version>
-    <aws-java-sdk.version>1.7.5</aws-java-sdk.version>
+    <aws-java-sdk.version>1.11.578</aws-java-sdk.version>
```

Comment:
JFYI: in recent versions, `aws-java-sdk` has split into `aws-java-sdk-s3`, `aws-java-sdk-ec2`, ... Just `aws-java-sdk` is now a set of dependencies to all these sub packages.

It would be nice to release a newer version as-is *without* narrowing down to the subpackage like `aws-java-sdk-s3` for safe compatibility. But we may want to narrow it down later.

Reply:
noted. I'll take at another PR.